.github/workflows/pages.yaml

```diff
@@ -13,6 +13,10 @@ jobs:
         uses: actions/checkout@v4
         with:
           fetch-depth: 1000  # make sure to fetch the old commit we diff against
+          
+      - name: fetch geth release
+        run: |
+          git fetch origin go-ethereum/release/1.14.3
```

Comment:
We can use `actions/checkout` for this? It allows specifying different repos, and checking out into a subdirectory

Reply:
Only need to fetch the geth release branch to have it available on the action git as the base fork. Two options to do so:
1.  current way - fetching the branch
2. checkout to the geth release using `actions/checkout` first to fetch it and then `actions/checkout` back to main